dbms/src/Flash/Mpp/MPPTask.cpp

```diff
@@ -94,14 +94,12 @@ void MPPTask::abortReceivers()
     receiver_set->cancel();
 }
 
-void MPPTask::abortDataStreams(AbortType abort_type)
+void MPPTask::abortDataStreams()
```

Comment:
Seems now we can rename `abortDataStreams` to `abortExecutor`?

Reply:
yes, renamed.